pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[build-system]
+# Minimum requirements for the build system to execute
+requires = ["setuptools>45"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pastastore"
+version = "0.10.0"
```

Comment:
don't we want the same (dynamic) version here as Pastas? 
so:
dynamic = ["version"]

[tool.setuptools.dynamic]
version = { attr = "pastastore.version.__version__" }

Reply:
added with [881b9bb](https://github.com/pastas/pastastore/pull/80/commits/881b9bbecbf3a3f4ec77267631c5554e720f6001)